generator/search_index_builder.cpp

```diff
@@ -95,6 +96,30 @@ class SynonymsHolder
   }
 };
 
+void GetCategoryTypes(CategoriesHolder const & categories, pair<int, int> const & scaleRange,
+                      feature::TypesHolder const & types, vector<uint32_t> & result)
+{
+  Classificator const & c = classif();
+
+  for (uint32_t t : types)
+  {
+    // Leave only 2 levels of types - for example, do not distinguish:
+    // highway-primary-bridge or amenity-parking-fee.
```

Comment:
Am I right that TruncValue(highway-primary-bridge, 2), will return highway-primary?